app/views/works/_curator_controlled_display.html.erb

```diff
@@ -1,35 +1,37 @@
 <div class="section-title">
-  <!-- DOI field (mostly for legacy DSpace works) -->
+  <!-- DOI field -->
   <div class="field">
-    DOI <br>
-    <%= @work.doi %>
+    DOI:<br>
+    <input class="text-not-editable" type="text" id="doi" name="doi" value="<%= @work.resource.doi %>" readonly />
```

Comment:
```suggestion
    <input id="doi" name="doi" value="<%= @work.resource.doi %>" readonly />
```

This class is being consistently applied with `readonly`, so I think just `readonly` would suffice? There's one CSS rule to update, but I think that's it. (See earlier comment.)

`text` is the default, so we could leave it off, I think?

(Same comment for all the inputs below.)